apps/juicebox_stream/test/juicebox_stream/stream/control_test.exs

```diff
@@ -11,9 +11,9 @@ defmodule JuiceboxStream.Stream.ControlTests do
         timer: nil
       },
       videos: {
-        %{ track_id: 1, votes: 0 },
-        %{ track_id: 2, votes: 0 },
-        %{ track_id: 3, votes: 0 }
+        %{ video: %{ video_id: 1 }, votes: 0 },
+        %{ video: %{ video_id: 2 }, votes: 0 },
+        %{ video: %{ video_id: 3 }, votes: 0 }
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.